lib/vcs.ml

```diff
@@ -64,28 +64,49 @@ let find_git () =
       | dir, (_, `Exited 0) -> Ok (Some (v `Git git ~dir:Fpath.(v dir)))
       | _ -> Ok None )
 
-let cmd_error cmd = function
-  | `Exited c ->
-      R.error_msgf "The following command exited with code %d:@\n%a" c Cmd.pp
-        cmd
-  | `Signaled c ->
-      R.error_msgf "The following command exited with signal %d:@\n%a" c Cmd.pp
-        cmd
+let cmd_error cmd ?err_msg =
+  match err_msg with
+  | None | Some "" -> (
+      function
```

Comment:
I think this doesn't read very well. What would you think about explicitly adding the extra `status` argument above and wrap this in a single pattern-matching?

Reply:
Yes, that indeed looks better :)